.github/workflows/scanning.yml

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2020, 2021 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Scan repository and containers for security vulnerabilities, secrets and
+# misconfigurations.
+
+name: Vulnerability and Misconfiguration Scan
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Run vulnerability scanner in repository mode
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'fs'
+          security-checks: 'vuln,secret,config'
+          ignore-unfixed: true
+          format: 'sarif'
+          output: 'trivy-results.sarif'
+          trivyignores: '.trivyignore'
```

Comment:
I have run trivy in my local workspace against the current code base. It has reported that we are using org.apache.logging.log4j:log4j-api in version 2.16.0. However, the Maven build will resolve log4j-api to version 2.18.0 based on Quarkus' dependency management configuration. It looks like trivy doesn't consider Maven's dependency resolution mechanism correctly but simply takes any values from pom.xml files for face value.
IMHO Trivy's (Maven) dependency vulnerability scanning doesn't seem to be very accurate, i.e. it seems to produce false positives ...

Reply:
This seems indeed the case, I'll probably raise an issue with trivy to get this fixed because we'll probably be affected too.
However, if this gets fixed, this will probably grant a great metric of how vulnerable dependencies of Hono are.

Reply:
Yes, however, until then we should remove the check FMPOV

Reply:
I can agree with that point of view, as the container image scanning from trivy also searches for these dependencies in the container itself. In there they are correctly found (because they exist as .jar file), and thus correctly scanned. I will disable the scanner.

Reply:
FMPOV we should remove it completely. Or do we get anything else out of the fs scan that we don't get out of scanning the images?

Reply:
Except for detecting of misconfigurations, which are not applicable for this repo, no.